src/screens/Home/surveyList.js

```diff
@@ -1,53 +1,102 @@
+  };
+  const slickConfig = { ...slickSettings, ...sliderEvents };
+
   useEffect(() => {
     window.addEventListener('wheel', (e) => {
       slideList(e.wheelDelta);
-    })
-  }, []);
+    });
+
+    dispatch({
+      type: 'SURVEY',
+      payload: currentBackground,
+    });
+  }, [currentBackground, dispatch]);
+
+  const beforeSlideChange = (next) => {
+    setCurrentBackground(list[next].attributes.cover_image_url);
+  };
+
+  const afterSlideChange = () => {
+    if (currentPage < total_pages) {
+      getList();
+    }
+  };
+
+  const getList = async () => {
+    const next_page = currentPage + 1;
+    try {
+      await Api.get(`api/v1/surveys?page[number]=${next_page}`, {
+        headers: {
+          Authorization: authContext.state.authorization_token,
+        },
+      }).then(function (response) {
+        if (response.status === 200) {
+          const new_list = [...list, ...response.data.data];
+          setList(new_list);
+          setCurrentPage(response.data.meta.page);
+        }
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
 
   const slideList = (wheel) => {
-    wheel > 0 ? (
-      sliderRef.current.slickNext()
-    ) : (
-      sliderRef.current.slickPrev()
-    )
-  }
+    wheel > 0 ? sliderRef.current.slickNext() : sliderRef.current.slickPrev();
+  };
 
   return (
     <div className="survey-list">
-      <Slider { ...slickSettings } ref={ sliderRef }>
-        <div>
-          <img src={ testSurvey } alt="test" className="survey-list__image" />
-        </div>
-        <div>
-          <img src={ testSurvey } alt="test" className="survey-list__image" />
-        </div>
-        <div>
-          <img src={ testSurvey } alt="test" className="survey-list__image" />
-        </div>
+      <Slider { ...slickConfig } ref={ sliderRef }>
+        { list.map(function (slide) {
+          return (
+            <div key={ slide.id }>
+              <img
+                src={ slide.attributes.cover_image_url }
+                alt={ slide.attributes.title }
+                className="survey-list__image"
+              />
+              <h3 className="survey-list__title">{ slide.attributes.title }</h3>
+              <p className="survey-list__description">
+                { slide.attributes.description }
+              </p>
+            </div>
+          );
+        }) }
       </Slider>
 
-      <div className="survey-list__information">
-        <div>
-          <h3 className="survey-list__title">Working from home Check-In</h3>
-          <p className="survey-list__description">
-            We would like to know about how you feel about our work from home.
-          </p>
-        </div>
-        <div className="survey-list__navigation">
-          <button className="button button--circle">
-            <span className="survey-list__detail-arrow" />
-          </button>
-        </div>
+      <div className="survey-list__navigation">
+        <button className="button button--circle">
```

Comment:
@rafayet-monon seems like the arrow button doesn't align with the title and description, how about moving the button into the slider too?
 
![Screen Shot 2563-11-12 at 11 01 46](https://user-images.githubusercontent.com/1772999/98894265-01b2dd80-24d7-11eb-8549-dc8afb5f0475.png)

![Screen Shot 2563-11-12 at 11 07 14](https://user-images.githubusercontent.com/1772999/98894373-40489800-24d7-11eb-8f05-83cdcae3f928.png)

Reply:
Fixed in [3ca95b5](https://github.com/rafayet-monon/surveyor/pull/17/commits/3ca95b5c9d5dd0050b6f841b99c76f5de4eb05a1)